hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/management/dstable/DistributionTable.java

```diff
@@ -165,16 +172,12 @@ private boolean checkAndHandleIfVisibleDsSwitchesFromCompleteToIncomplete(
     /**
      * DistributionTableFilterEvent.
      *
-     * @param event
+     * @param filterEvent
      *            as instance of {@link DistributionTableFilterEvent}
      */
-    @EventBusListenerMethod(scope = EventScope.UI)
-    void onEvent(final DistributionTableFilterEvent event) {
-        if (event == DistributionTableFilterEvent.FILTER_BY_TEXT
-                || event == DistributionTableFilterEvent.REMOVE_FILTER_BY_TEXT
-                || event == DistributionTableFilterEvent.FILTER_BY_TAG) {
-            UI.getCurrent().access(this::refreshFilter);
-        }
+    @EventBusListenerMethod(scope = EventScope.UI, filter = DeploymentViewFilter.class)
```

Comment:
Better rename all the filter classes to `OnlyEventsFrom*Filter` or even `AllEventsFrom*View`. Than it is more obvious for what you are filterting here => 1. all events comming from view "x" AND  `DistributionTableFilterEvent`.